jetbrains-rider/tst/software/aws/toolkits/jetbrains/services/lambda/completion/DotNetHandlerCompletionProviderTest.kt

```diff
@@ -11,7 +11,7 @@ import software.aws.toolkits.core.lambda.LambdaRuntime
 
 class DotNetHandlerCompletionProviderTest : AwsReuseSolutionTestBase() {
 
-    override fun getSolutionDirectoryName(): String = "SamHelloWorldApp"
+    override fun getSolutionDirectoryName(): String = "testData/solutions/SamHelloWorldApp/SamHelloWorldApp.sln"
```

Comment:
The SamHelloWorldApp directory name will be required for previous versions before 232, correct?